audformat/core/utils.py

```diff
@@ -376,6 +360,43 @@ def join_labels(
     return joined_labels
 
 
+def join_schemes(
+        dbs: typing.Sequence[Database],
+        scheme: str,
```

Comment:
Should we rename to `scheme_id` so that it is more obvious that not a `Scheme` object but an ID is expected?

Reply:
Good idea, I renamed it to `scheme_id`